### PR TITLE
regen from latest boms, include gitops parallelism

### PR DESCRIPTION
**Pull request by triceam**
Signed-off-by: Andrew Trice <amtrice@us.ibm.com>

### CODE REVIEW

File: apply-all.sh [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
   while read dir;
 do
   name=$(echo "$dir" | sed -E "s~[.]/(.*)~\1~g")
 
+  TYPE=$(grep "deployment-type/gitops" ./${name}/bom.yaml | sed -E "s~[^:]+: [\"'](.*)[\"']~\1~g")
+
+  if [[ "${TYPE}" == "true" ]]; then
+    PARALLELISM=3
+    echo "***** Setting parallelism for gitops type deployment for step ${name} to ${PARALLELISM} *****"
+    continue
```
Comment (timroster) [inline apply-all.sh:21]: @triceam - I don't think you want `continue` here as it seems to skip to the next loop item Prevents 220-dev-tools from installing. Output from `apply-all`:

```

Apply complete! Resources: 23 added, 0 changed, 0 destroyed.
***** Setting parallelism for gitops type deployment for step 220-dev-tools to 3 *****
/workspaces/current $
```

Comment (triceam) [reply apply-all.sh:21]: good catch, thanks.  that continue should have been removed. this was leftover form a copy/paste

File: apply-all.sh [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
+  fi
+
+  OPTIONAL=$(grep "apply-all/optional" ./${name}/bom.yaml | sed -E "s~[^:]+: [\"'](.*)[\"']~\1~g")
+
+  if [[ "${OPTIONAL}" == "true" ]]; then
+    echo "***** Skipping optional step ${name} *****"
+    continue
```
[the remark this answers is not in the document]
Comment (triceam) [reply apply-all.sh:28]: this one is intentional b/c it is skipping the optional BOMs.  `continue` is the correct operation in this case.

[the remark this answers is not in the document]
Comment (timroster) [reply apply-all.sh:28]: makes sense here then!

**Comment by triceam** [comment]
Closing this PR in favor of https://github.com/IBM/automation-ibmcloud-infra-openshift/pull/13

**Pull request closed without merging**